dom.bs

```diff
@@ -4860,6 +4862,14 @@ null if <var>event</var>'s {{Event/type}} attribute value is "<code>load</code>"
 <a>document</a> does not have a <a for=Document>browsing context</a>; otherwise the
 <a>document</a>'s <a>relevant global object</a>.
 
+<p>A <a>document</a>'s <dfn export id=concept-document-set-the-mode>set the mode</dfn> algorithm,
+given a <var>new mode</var>, performs the following steps:
+
+<ol>
+ <li><p>If <a>this</a>'s <a for=Document>mode locked</a> is true, return.
```

Comment:
then return*

Add a newline here as well to separate from the following `<li>`.